articles/quickstart/webapp/java-spring-security-mvc/_includes/_login.md

```diff
@@ -175,38 +264,27 @@ private void invalidateSession(HttpServletRequest request) {
         request.getSession().invalidate();
     }
 }
-
-## Display the Home Page
-
-Now that the user is authenticated (the tokens exists), the framework will allow them to access our protected resources. In the `HomeController` you can get the existing Authentication object and even the Principal that represents it. Let's set that as the `userId` attribute so it can be used from the JSP code:
-
-```java
-// src/main/java/com/auth0/example/mvc/HomeController.java
-
-@RequestMapping(value = "/portal/home", method = RequestMethod.GET)
-protected String home(final Map<String, Object> model, final Principal principal) {
-    if (principal == null) {
-        return "redirect:/logout";
-    }
-    model.put("userId", principal);
-    return "home";
-}
 ```
 
 ## Run the Sample
 
-To run the sample from a terminal, change the directory to the root folder of the project and execute the following line:
+To build an run the project, use the command:
 
 ```bash
 ./gradlew clean bootRun
 ```
 
-After a few seconds, the application will be accessible on `http://localhost:3000/`. Try to access the protected resource [http://localhost:3000/portal/home](http://localhost:3000/portal/home) and note how you're redirected by the framework to the login page. The widget displays all the social and database connections that you have defined for this application in the [dashboard](${manage_url}/#/).
+Or if you are using Windows:
+
+```bash
+gradlew.cmd clean bootRun
+```
+
 
-![Login using Lock](/media/articles/java/login-with-lock.png)
+Point your browser to [http://localhost:3000](http://localhost:3000). Follow the **Log In** link to log in or sign up to your Auth0 tenant. 
 
-After a successful authentication you'll be able to see the home page contents.
+![login page](/media/articles/web/hosted-login.png)
```

Comment:
did this screenshot change?

Reply:
Previously the QS was using `login-with-lock.png`. I changed it to use `hosted-login.png` (it's what the updated Vue.js QS uses). Is it not correct?